commit 34c11a481baafd9824ef681b6d359ca34ab04ad8
Author: Vishesh Handa <me@vhanda.in>
Date:   Wed May 6 14:27:37 2015 +0200

    Baloo Runner: Use the filepath as the unique id of the match
    
    Baloo no longer exports the identifier of the file

diff --git a/runners/baloo/baloosearchrunner.cpp b/runners/baloo/baloosearchrunner.cpp
index 0f5e066..2bbd7c1 100644
--- a/runners/baloo/baloosearchrunner.cpp
+++ b/runners/baloo/baloosearchrunner.cpp
@@ -114,7 +114,7 @@ QList<Plasma::QueryMatch> SearchRunner::match(Plasma::RunnerContext& context, co
 
         QString iconName = QMimeDatabase().mimeTypeForFile(localUrl).iconName();
         match.setIcon(QIcon::fromTheme(iconName));
-        match.setId(it.id());
+        match.setId(it.filePath());
         match.setText(url.fileName());
         match.setData(url);
         match.setType(Plasma::QueryMatch::PossibleMatch);
